update several statements (#2831)
pingcap/docs: sql-statements/sql-statement-show-variables.md

## Examples

> List all TiDB specific variables. For the meaning of them, refer to [TiDB Specific System Variables](/tidb-specific-system-variables.md).

Reviewer: ```suggestion
List all TiDB specific variables. For detailed description, refer to [TiDB Specific System Variables](/tidb-specific-system-variables.md).
```
